src/mixins/name-request-mixin.ts

```diff
@@ -1,92 +1,48 @@
-// Libraries
+import Vue from 'vue'
```

Comment:
This file is now exactly the same as in Filings UI... in preparation for moving it to shared components library.